README.md

```diff
@@ -1 +1,24 @@
-# Hello_microverse-project
\ No newline at end of file
+# Hello_microverse-project
+
+# Hello microverse
+In this project, we will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices we learned about in previous steps. we will be using them in all Microverse projects.
+
+# Built With
+HTML 
+CSS
+
+# Getting Started
+The prerequisites was only to complete the previous lessons so that we amy have the general idea of what we will be doing in this exercise
+
+# Authors
+christian biringanine
+
+GitHub: [hander]github.com/christianbiring1
+Twitter: [handler]twitter.com/christianbiringan4
+LinkedIn: LinkedIn
```

Comment:
- [ ] [OPTIONAL] Kindly fix your social links. You can do that using this syntax `[handler](link)` the handler has to be wrapped with square brackets and next to it the link has to be wrapped in parenthesis so that it will eventually look like this: 
```
- GitHub: [christianbiring1](https://github.com/christianbiring1)
- Twitter: [christianbiringan4](https://twitter.com/christianbiringan4)
```
which would look like this:

- GitHub: [christianbiring1](https://github.com/christianbiring1)
- Twitter: [christianbiringan4](https://twitter.com/christianbiringan4)